io/plugins/eu.esdihumboldt.hale.io.csv/plugin.xml

```diff
@@ -53,9 +53,9 @@
                ref="eu.esdihumboldt.hale.io.csv">
          </contentType>
          <providerParameter
-               description="States if the first line in the CSV file should be skipped (e.g. because it contains the header)"
-               label="Skip first line"
-               name="skip"
+               description="States how many lines should be skipped when reading instances from a CSV file (e.g. because of the header)"
+               label="Skip N lines"
+               name="skipN"
```

Comment:
Regarding the behavior you chose now to replace the existing parameter and be backwards compatible on it. So the name needs to stay `skip`.

Maybe mention that boolean values are also supported for backwards compatibility and what their effect is?